modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceAccelerators.path

```diff
@@ -337,6 +337,17 @@
 	<td>//div[contains(@class,'add-to-cart d-flex my-2 pt-5')]//a[contains(@href,'${key_productName}')]</td>
     <td></td>
 </tr>
+<!--COMPARE-->
+<tr>
+	<td>CHECK_ITEM_COMPARE</td>
+	<td>//div[contains(@class,'mini-compare active')]//div[contains(@style,'${key_productName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>REMOVE_ITEM_COMPARE</td>
+	<td>//div[contains(@class,'mini-compare active')]//div[contains(@style,'${key_productName}')]/../../..//button[contains(@class,'mini-compare-delete btn btn-monospaced btn-sm btn-unstyled')]</td>
+	<td></td>
+</tr>
```

Comment:
Since this locator is not being used anywhere on the test it can be removed from the PR.